packages/cli/scripts/preinstall.js

```diff
@@ -1,21 +1,40 @@
+const process = require('node:process')
 // @ts-check
 const { drawBox } = require('@prisma/internals/dist/utils/drawBox')
 
 export function main() {
-  // process.version (e.g. `v16.0.0`)
-  printMessageAndExitIfUnsupportedNodeVersion(process.version)
+  const nodeVersion = process.version.slice(1)
```

Comment:
```suggestion
  // process.version (e.g. `v20.0.0`) starts with `v`, we remove it.
  const nodeVersion = process.version.slice(1)
```
Keep the example it's nice to have it here